ci/compiler/pio.py

```
@@ -621,6 +667,17 @@ def _build_with_fbuild(self, example: str) -> SketchResult:
             green_color = "\033[32m"
             reset_color = "\033[0m"
             print(f"{green_color}SUCCESS: {example}{reset_color}")
+            # Mirror top-level fbuild artifacts to the legacy .pio/build path so
+            # downstream tooling (QEMU/avr8js workflows, crash decoders) keeps
+            # working without having to know about fbuild's layout.
+            try:
+                _mirror_fbuild_artifacts_to_pio(self.build_dir, environment)
+            except Exception as mirror_err:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)